infra/concourse/build/developer-tools/Dockerfile

```diff
@@ -73,6 +85,15 @@ RUN /build/install_terraform_docs.sh ${TERRAFORM_DOCS_VERSION}
 ADD ./build/install_bats.sh /build/
 RUN /build/install_bats.sh ${BATS_VERSION}
 
+ADD ./build/install_bats_support.sh /build/
```

Comment:
Nit: Could we just combine these into a single `install_bats.sh` script?

Reply:
Good point, addressed.